proposals/003_Clarify-Nullable.md

```diff
@@ -8,7 +8,7 @@
 |Proposal |[003](https://github.com/OAI/OpenAPI-Specification/tree/master/proposals/003_Clarify-Nullable.md)|
 |Authors|[Ted Epstein](https://github.com/tedepstein)|
 |Review Manager |TBD|
-|Status |Proposal|
```

Comment:
According to the template, the statuses are:

> Proposal, Draft, Promoted, or Abandoned

Reply:
@MikeRalphson , thanks. I've made the change.